test_plans/ceph-with-dash-lxd.yaml

```diff
@@ -0,0 +1,5 @@
+bundle: bundle:~juju-qa/ceph-with-dash
+bundle_name: ceph-with-dash-lxd
+bundle_file: ceph-with-dash-lxd.yaml
```

Comment:
Please remove `bundle_file: ceph-with-dash-lxd.yaml`. This bundle doesn't need bundle_file field.